tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/exception/TMCConnectionException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.tajo.catalog.exception;
+
+import org.apache.tajo.error.Errors.ResultCode;
+import org.apache.tajo.exception.TajoError;
+
+/**
+ * Tajo Metadata Connector's connection error
+ */
+public class TMCConnectionException extends TajoError {
```

Comment:
IMHO, it is better if the class name is intuitive even though its length is quite long. 
In addition, ```TM``` is usually used for ```TajoMaster``` in our architecture. I think that this name makes a lot of confusion.